client/layout/index.jsx

```diff
@@ -123,3 +117,13 @@ module.exports = React.createClass( {
 		);
 	}
 } );
+
+export default connect(
+	( state ) => {
+		return {
```

Comment:
if you did `const { isLoading, section, noSidebar } = state.ui;` you could then simplify the object literal to `{ isLoading, section, noSidebar }`.

No necessary, I just find it easier to read.